projects/Gancio/default.nix

```
@@ -0,0 +1,24 @@
+{
+  pkgs,
+  lib,
+  sources,
+}@args:
+{
+  metadata = {
+    summary = "Gancio Shared agenda for local communities that supports Activity";
+      subgrants = [
+        "gancio"
+        "hex designs"
+      ];
+  };
+  nixos.modules.programs = {
```

Gancio is actually a service in nixpkgs and is not a program (see [module](https://search.nixos.org/options?channel=unstable&size=50&sort=relevance&type=packages&query=gancio))

```suggestion
  nixos.modules.services = {
```